client/.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
Why are there node_modules?

Reply:
ESLint and Stylelint are Node.js programs. You need to install them via NPM in order to run them.

Reply:
If you're worried about this making things complicated, don't be. I explained everything in the documentation I wrote in #3, and it's pretty straightforward.

Reply:
Resolving since it looks like this helps you lint locally.